wix.md

```diff
@@ -0,0 +1,38 @@
+# Wix Sites integration
+
+You can use Wikipedia Preview in your [Wix Site](https://www.wix.com/) if you have any premium account by [embedding custom code to your site](https://support.wix.com/en/article/embedding-custom-code-to-your-site). Follow these simple instructions to get started and bring the sum of all knowledge to your readers.
+
+First head to your site's [dashboard settings](https://www.wix.com/my-account/site-selector/?buttonText=Open%20Settings&title=Select%20a%20Site&autoSelectOnSingleSite=true&actionUrl=https://www.wix.com/dashboard/{{metaSiteId}}/settings) to load and initialize Wikipedia Preview:
+
+To load:
+  1. Click the Custom Code tab under Advanced Settings
+  2. Click either "Add Custom Code" at the top right or the "Add Code" button in the "Head" box
```

Comment:
This file should be loaded at the end of the body so it doesn't slow down initial rendering. It looks like both load and init can be included in the same code block for simplicity.

Reply:
I like that simplicity. Although given that we need to "Load code on each new page" for `init` to trigger, if we include both in the same code block I'm seeing the `unpkg` get added many times in the HTML. Is that acceptable?

<img width="480" alt="image" src="https://user-images.githubusercontent.com/4752599/113866238-13916780-977b-11eb-9184-41185ae2b456.png">

Reply:
Not ideal. Let's create 2 blocks but both at the end of the body.